app/ui/privacy.js

```diff
@@ -98,9 +63,6 @@ module.exports = function(state, emit) {
                 </li>
               </ul>
             </li>
-            <li class="mb-4">
```

Comment:
Let's put this section back and specify that we use DigitalOcean for hosting.

Reply:
Fair enough, I considered that but we should also remember to add back anything else we use as well (e.g. any cloud storage provider)